### PR TITLE
 chore: skip ChatGPT review in BranchMerge

### PR DESCRIPTION
**Pull request by li-jia-nan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [x] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | fix: stop ChatGPT review in BranchMerge |
| 🇨🇳 Chinese | 在合并分支的时候关闭 ChatGPT review |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: .github/workflows/chatgpt-cr.yml [hunk 1]
```diff
@@ -11,6 +11,7 @@ on:
 jobs:
   test:
     runs-on: ubuntu-latest
+    if: github.event.pull_request.head.ref != 'feature' && github.event.pull_request.head.ref != 'master'
     steps:
       - uses: anc95/ChatGPT-CodeReview@main
         env:
```
[the remark this answers is not in the document]
Comment (afc163) [reply .github/workflows/chatgpt-cr.yml:17]: 这样不行，源分支是 master 或 feature 就跳过了，哪怕是正常的 PR

[the remark this answers is not in the document]
Comment (li-jia-nan) [reply .github/workflows/chatgpt-cr.yml:17]: 正常的 PR 源分支不会是 master 或 feature 吧

[the remark this answers is not in the document]
Comment (afc163) [reply .github/workflows/chatgpt-cr.yml:17]: github.event.pull_request.head 是 source branch？哦好像是对的

[the remark this answers is not in the document]
Comment (afc163) [reply .github/workflows/chatgpt-cr.yml:17]: 补一下这里的分支名 https://github.com/ant-design/ant-design/blob/master/.github/workflows/pr-check-ci.yml#L24

**Review by afc163 — APPROVED** [review]

File: .github/workflows/chatgpt-cr.yml [hunk 2]
```diff
@@ -11,6 +11,7 @@ on:
 jobs:
   test:
     runs-on: ubuntu-latest
+    if: github.event.pull_request.head.ref != 'feature' && github.event.pull_request.head.ref != 'master' && github.event.pull_request.head.ref != 'next' && github.event.pull_request.head.ref != 'master-merge-feature' && github.event.pull_request.head.ref != 'feature-merge-master' && github.event.pull_request.head.ref != 'next-merge-master' && github.event.pull_request.head.ref != 'next-merge-feature'
```
Comment (afc163) [inline .github/workflows/chatgpt-cr.yml:14]: 想了想，是不是改成

```
if: github.event.pull_request.head.ref == 'feature' || github.event.pull_request.head.ref == 'master'  || github.event.pull_request.head.ref == '4x-stable'
```

这样就够用了。

**Pull request merged**